x-pack/test/fleet_api_integration/apis/outputs/crud.ts

```diff
@@ -99,6 +101,56 @@ export default function (providerContext: FtrProviderContext) {
         });
       });
 
+      it('should allow to create a logstash output ', async function () {
+        const { body: postResponse } = await supertest
+          .post(`/api/fleet/outputs`)
+          .set('kbn-xsrf', 'xxxx')
+          .send({
+            name: 'My Logstash Output',
+            type: 'logstash',
+            hosts: ['test.fr:443'],
+            ssl: {
+              certificate: 'CERTIFICATE',
+              key: 'KEY',
+              certificate_authorities: ['CA1', 'CA2'],
+            },
+          })
+          .expect(200);
+
+        const { id: _, ...itemWithoutId } = postResponse.item;
+        expect(itemWithoutId).to.eql({
+          name: 'My Logstash Output',
+          type: 'logstash',
+          hosts: ['test.fr:443'],
+          is_default: false,
+          is_default_monitoring: false,
+          ssl: {
+            certificate: 'CERTIFICATE',
+            key: 'KEY',
+            certificate_authorities: ['CA1', 'CA2'],
+          },
+        });
+      });
+
+      it('should not allow to create a logstash output with http hosts ', async function () {
+        const { body: postResponse } = await supertest
+          .post(`/api/fleet/outputs`)
+          .set('kbn-xsrf', 'xxxx')
+          .send({
+            name: 'My Logstash Output',
+            type: 'logstash',
+            hosts: ['https://test.fr:443'],
+            ssl: {
+              certificate: 'CERTIFICATE',
+              key: 'KEY',
+              certificate_authorities: ['CA1', 'CA2'],
+            },
+          })
+          .expect(400);
+
+        expect(postResponse.message).match(/Invalid logstash host/);
```

Comment:
how come the error message is not `'Invalid logstash host should not start with http(s)'` ? It should match the regex here right?
https://github.com/elastic/kibana/blob/d59b27a0af928ab50843c7cef17df5a9eb3b4478/x-pack/plugins/fleet/server/types/models/output.ts#L13-L14